src/main/java/com/cburch/logisim/fpga/hdlgenerator/Hdl.java

```diff
@@ -112,6 +167,67 @@ public static String xorOperator() {
     return isVhdl() ? Vhdl.getVhdlKeyword(" XOR ") : "^";
   }
 
+  public static String addOperator(String signalOne, String signalTwo, boolean signed) {
+    return (isVhdl() ? "std_logic_vector(" : "")
+            + typecast(signalOne, signed)
+            + " + "
+            + typecast(signalTwo, signed)
+            + (isVhdl() ? ")" : "");
```

Comment:
so this ternary is in separate line, brackts could be removed.

Reply:
They can't be removed.
```
Hdl.java:175: error: incompatible types: String cannot be converted to boolean
            + isVhdl() ? ")" : "";
            ^
```